app/assets/javascripts/welcome.coffee

```diff
@@ -35,3 +35,26 @@ $ ->
       success: (data) ->
         $('#partial_news').html data
     event.preventDefault()
+
+  $("#clean_search_field").click ->
+    $('#search').val("")
+    form = $(".search_form")
+    formData = form.serialize()
+    $.ajax
+      type: 'GET'
+      data: formData
+      dataType: 'html'
+      success: (data) ->
+
+        $('#partial_news').html data
+
+  $('body').on 'click', '#tiding_table_pagination  a', (event, data, status, xhr) ->
+    url = $(this).attr("href")
+    $.ajax
+      url: url
+      type: 'GET'
+      dataType: 'html'
+      success: (data) ->
+        debugger;
+        $('#partial_news').html data
+    event.preventDefault()
```

Comment:
add to sublime setting `"ensure_newline_at_eof_on_save": true` to avoiding :no_entry_sign:  sign